src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -60,14 +81,41 @@ func configureLogging() *rotator.SizeRotator {
 		log.Debugf("No logglyToken, not sending error logs to Loggly")
 	} else {
 		log.Debugf("Sending error logs to Loggly")
-		remoteWriter := logglyErrorWriter{loggly.New(logglyToken)}
-		errorOut = NonStopWriter(errorOut, remoteWriter)
+		logglyClient = &logglyErrorWriter{loggly.New(logglyToken)}
```

Comment:
Since loggly is initialized right at the beginning, but we don't get a proxied HTTP client until the call to Configure(), this theoretically means that some stuff could be sent to Loggly via an unproxied HTTP client, right?